docs/reference/logging/journald.md

```diff
@@ -41,6 +42,11 @@ You can set the logging driver for a specific container by using the
 Users can use the `--log-opt NAME=VALUE` flag to specify additional
 journald logging driver options.
 
+### tag
+
+Template to set `CONTAINER_TAG` value in journald logs, refer to
```

Comment:
Take a look at other splunk option descriptions, they are not always sentences. "Some markup" sounds ambiguous too.

I can change it to: "Specify template to set `CONTAINER_TAG` value in journald logs. Refer to
[log tag option documentation](log_tags.md) for customizing the log tag format."